torch/jit/frontend.py

```diff
@@ -281,6 +282,10 @@ def _forward(self):
         for arg in fn_def.args.args + fn_def.args.kwonlyargs:
             # Replace potentially unsupported type annotations by "Any"
             arg.annotation = unused_def.args.args[0].annotation
+        if _is_drop_fn(fn):
+            # Dropping potentially unsupported return type annotation for jit._drop
+            fn_def.returns = None
+            fn_def.type_comment = None
```

Comment:
I don't think this is needed, because we won't call get_jit_def on a method if it's marked with _drop.